.ddev/config.yaml

```diff
@@ -28,6 +28,9 @@ web_environment:
   - ADMIN_PASS=admin
   - ADMIN_EMAIL=admin@example.com
   - WC_VERSION=10.1.2
+  - WP_DEBUG=true
```

Comment:
wasn't it already enabled here?

https://github.com/woocommerce/woocommerce-paypal-payments/blob/b472faa837a0e021dc2e1c4677ea37df010b3c35/.ddev/commands/web/orchestrate.d/05_wp_config.sh#L18